test/unit_test/static/enzimology_test.cpp

```diff
@@ -0,0 +1,37 @@
+// The MIT License (MIT)
+//
+// Copyright (c) 2023 Meskauskas Audrius
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+
+#include <mp-units/systems/enzimology/enzimology.h>
+
+namespace {
+
+using namespace mp_units;
+
+// Check agreements between international unit (IU), unit and katal
+static_assert(1 * enzimology::unit == 1 * enzimology::iu);
+static_assert(1 * enzimology::unit < 16.6667 * enzimology::nanokatal);
+static_assert(1 * enzimology::unit > 16.6666 * enzimology::nanokatal);
+static_assert(1 * enzimology::katal == 1'000'000'000 * enzimology::nanokatal);
+
```

Comment:
You may want to add some tests for quantity_specs as well similar to [isq_test.cpp](https://github.com/mpusz/mp-units/blob/master/test/unit_test/static/isq_test.cpp).